src/mono/wasm/templates/templates/console/console.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
we already have the `console.0.csproj`, so the `console.csproj` is not needed.

Reply:
we might need `<AllowUnsafeBlocks>true</AllowUnsafeBlocks>` in the `console.0.csproj`. @pavelsavara, are the unsafe blocks always needed with the new interop? I noticed you added it to the browser template.